docs/python_docs/python/tutorials/packages/symbol/data.md

```diff
@@ -0,0 +1,474 @@
+<!--- Licensed to the Apache Software Foundation (ASF) under one -->
+<!--- or more contributor license agreements.  See the NOTICE file -->
+<!--- distributed with this work for additional information -->
+<!--- regarding copyright ownership.  The ASF licenses this file -->
```

Comment:
I think we should get rid of Symbol tutorials

Reply:
I don't disagree. Let's do that separately though. Other people might still be using that.